### PR TITLE
feat(automl): add AutoML as preview feature

### PR DESCRIPTION
**Pull request by kuroidoruido**

### CODE REVIEW

File: src/client/Trainables.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { AutoMlTrainable } from './automl/AutoMlTrainable';
+import { GladiaClientParams } from './gladia-client-params';
+
+export class Trainables {
+  private readonly autoMlInst: AutoMlTrainable;
+
+  constructor(params: GladiaClientParams) {
+    this.autoMlInst = new AutoMlTrainable(params);
```
Comment (jsoto-gladia) [inline src/client/Trainables.ts:8]: autoML naming with capital L ?

File: src/client/automl/AutoMlTrainable.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  ): Promise<AutoMlPredictOutput<TResult[]>> {
+    if (!isPreviewEnabled(this.params) || isNotDefined(this.params.autoMlBaseUrl)) {
+      throw new PreviewFeatureError();
+    }
+    const body: AutoMlPredictRequestBody<TData> = {
+      model_id: args.model_id,
+      data: args.data,
+    };
+    return this.httpClient.post({
+      url: '/tabular/predict/',
```
Comment (jsoto-gladia) [inline src/client/automl/AutoMlTrainable.ts:59]: Use multimodal by default please

File: src/client/automl/AutoMlTrainable.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    };
+    if (isDefined(args.eval_metric)) {
+      body.eval_metric = args.eval_metric;
+    }
+    if (isDefined(args.time_limit)) {
+      body.time_limit = args.time_limit;
+    }
+
+    return this.httpClient.post({
+      url: '/tabular/create-model/',
```
Comment (jsoto-gladia) [inline src/client/automl/AutoMlTrainable.ts:38]: Miltimodal please ?

**Review by jsoto-gladia — CHANGES_REQUESTED** [review]
Can tou allow me to change the path of the url ? IE if I want to change text/create-model or tabular/create model or text/prediction or tabular/prediction

**Comment by kuroidoruido** [comment]
> Can tou allow me to change the path of the url ? IE if I want to change text/create-model or tabular/create model or text/prediction or tabular/prediction

I added a kind input parameter to specify the kind of train/predict, that impact the url

**Review by jsoto-gladia — APPROVED** [review]

**Pull request merged**